File: Development/lab6/pokemon.py
import json

class Pokemon:
    def __init__ (self, id, name, weight, height, exp):
        self.id = id
        self.name = name
        self.weight = weight
        self.height = height
        self.base_exp = exp
    
    def toJSON (self):
        dict_val =  {
            'id':self.id,
            'name':self.name,
            'weight':self.weight,
            'height':self.height,
            'base_exp':self.base_exp
        }

        return json.dumps(dict_val)
    
    def toHTMLRow (self):
        str = "<tr><td>{0}</td><td>{1}</td><td>{2}</td><td>{3}</td><td>{4}</td>".format(self.id, self.name, self.weight, self.height, self.base_exp)
        return str

File: Development/lab6/Pokemon.js
class Pokemon {
    constructor (id, name, weight, height, exp) {
        this.id = id;
        this.name = name;
        this.weight = weight;
        this.height = height;
        this.base_exp = exp;
    }

    toJSON () {
        const o = {
            'id':this.id,
            'name':this.name,
            'weight':this.weight,
            'height':this.height,
            'base_exp':this.base_exp
        }
        return JSON.stringify(o);
    }

    toHTMLRow () {
        str = "<tr>";
        // this = Pokemon obj
        // i iterate through this[] list of Pokemon object attributes
        for (let i in this) {
            str += "<td>" + this[i] + "</td>";
        }
        str += "</tr>";
        return str;
    }
}

// To handle running on commandline
if (global && module && module.exports){
    console.log("Running pokemon.js via the command line");
    module.exports = {
        'Pokemon': Pokemon
    };
}

let p = new Pokemon (001, "Vayne", 60, 90, 560);
console.log(p.toHTMLRow());